controllers/binding/snapshotenvironmentbinding_adapter_test.go

```diff
@@ -318,23 +318,22 @@ var _ = Describe("Binding Adapter", Ordered, func() {
 		Expect(err == nil || errors.IsNotFound(err)).To(BeTrue())
 		err = k8sClient.Delete(ctx, hasSnapshot)
 		Expect(err == nil || errors.IsNotFound(err)).To(BeTrue())
+		err = k8sClient.Delete(ctx, deploymentTargetClaim) // Do not expect err to be nil. In some cases dtc is deleted during test
```

Comment:
The source test is reporting this:
```
Error: controllers/binding/snapshotenvironmentbinding_adapter_test.go:321:3: this value of err is never used (SA4006)
```
I think the solution is to add the `Expect(err == nil || errors.IsNotFound(err)).To(BeTrue())` below it.